tests/manifest-build/test_builds_successful.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import subprocess
+
+def kustomize_build(manifest_path):
+    return subprocess.call(f"kustomize build {manifest_path}".split())
+
+TO_ROOT = "../../"
+DEPLOYMENTS_FOLDER = TO_ROOT + "docs/deployment/"
+
+def test_vanilla():
```

Comment:
This might be better handled as part of unit test, @ryansteakley can this be a unit test? we just need to assert that the file is building. If so, we can keep this until we have that.

Seems too much to have a separate test suite just for this or should we merge it with e2e tests just for time being?

Reply:
moved to e2e test suite